demo.py

```diff
@@ -0,0 +1,252 @@
+
+    # Fix existing sites binary variables equal to 1
+    cqm.fix_variables({key[2]: 1.0 for key in tower_vars.keys()})
+
+    return cqm
+
+def visualize(region_map, existing_towers, new_locs, build_sites):
+    ''' Visualize the scenario and solution.
+    Args:
+        - region_map: gdf. Whole region map.
+        - existing_towers: df. Dataframe containing tower information.
+        - new_locs: list of [float, float]. New points as [lat, long].
+        - build_sites: list of [float, float]. Build site points as [lat, long].
+    Returns:
+        None.
+    '''
+
+    print("\nVisualizing scenario and solution...")
+
+    # Initialize figure and axes
+    _, (ax, ax_final) = plt.subplots(nrows=1, ncols=2, figsize=(32, 12))
+    ax.axis('off')
+    ax_final.axis('off')
+
+    # Draw borders or region
+    region_map.plot(ax = ax, color = '#d3d3d3', zorder=0)
+    region_map.plot(ax = ax_final, color = '#d3d3d3', zorder=0)
+
+    # Draw existing towers
+    gdf_towers = gpd.GeoDataFrame(
+        existing_towers, geometry=gpd.points_from_xy(existing_towers.Longitude, existing_towers.Latitude))
+    gdf_towers.plot(ax=ax, color='r', zorder=2)
+    gdf_towers.plot(ax=ax_final, color='r', zorder=2)
+
+    # Draw radius around existing towers
+    radius = 30
+    towers_radius = gdf_towers.copy()
+    towers_radius['geometry'] = towers_radius['geometry'].buffer(radius/111)
+    towers_radius.plot(ax=ax, color='r', alpha=0.1, zorder=1)
+    towers_radius.plot(ax=ax_final, color='r', alpha=0.1, zorder=1)
+
+    # Draw new potential build sites on map
+    new_locations = pd.DataFrame(new_locs, columns=['Latitude','Longitude'])
+    gdf_new = gpd.GeoDataFrame(
+        new_locations, geometry=gpd.points_from_xy(new_locations.Longitude, new_locations.Latitude))
+    gdf_new.plot(ax=ax, color='y', zorder=8)
+
+    # Draw new selected build sites on map
+    new_builds = pd.DataFrame(build_sites, columns=['Latitude','Longitude'])
+    gdf_builds = gpd.GeoDataFrame(
+        new_builds, geometry=gpd.points_from_xy(new_builds.Longitude, new_builds.Latitude))
+    gdf_builds.plot(ax=ax_final, color='b', zorder=8)
+
+    # Draw radius around selected build sites
+    build_radius = gdf_builds.copy()
+    build_radius['geometry'] = build_radius['geometry'].buffer(radius/111)
+    build_radius.plot(ax=ax_final, color='b', alpha=0.1, zorder=1)
+
+    # Make the figure look good
+    ax.set_title("Potential Sites", fontsize = 24)
+    ax_final.set_title("Determined Sites", fontsize = 24)
+
+    # Save the figure
+    plot_filename = 'map.png'
+    plt.savefig(plot_filename)
+    print("\nOutput saved as", plot_filename)
+
+
+if __name__ == "__main__":
+
+    # Load and draw country map from geojson file
+    print("\nLoading map and scenario...")
+    filename = "data/germany_states.geojson"
+    file = open(filename)
+    germany_map = gpd.read_file(file)
+
+    # Load existing towers
+    existing_towers = get_existing_towers("locations.txt")
```

Comment:
Looks like this should be:
```suggestion
    existing_towers = get_existing_towers("data/locations.txt")
```